### ISSUE TITLE
Channel-Network Tabs Support

### ISSUE THREAD

**Issue by zoltyvigo** [kiwiirc/kiwiirc#158]
In the old kiwiirc client, there is a configuration to change the appearance of the channel between tabs and list

![image](https://user-images.githubusercontent.com/6929524/33872194-27946334-df16-11e7-93e4-bf476aff3ae6.png)

In the new kiwiirc client it does not have it and I show the differences between the old and the new.
When it is embedded, the list reduces space for channel chat and becomes harder to read.

![image](https://user-images.githubusercontent.com/6929524/33872487-69a4edc4-df17-11e7-9356-83fc70c00a78.png)

Is it possible to implement the possibility of choosing between list and tab?

**Comment by Fudster** [comment]
I second this. I personally prefer the channels and PMS at the top as a tab.

**Comment by know75** [comment]
I third it. I like tabs on the side, but they're often just totally impractical on a smaller screen. Please allow users to have the choice as they even did in the previous version.

**Comment by zoltyvigo** [comment]
Any planned date for implementation? It's something that they ask me a lot.

**Comment by prawnsalad** [comment]
This is on the roadmap, but won't be a very soon thing. Hopefully in the next month or 2 we can get this in place

**Comment by prawnsalad** [comment]
Waiting on https://github.com/kiwiirc/kiwiirc/issues/903

**Comment by finalhit** [comment]
Any update on this? Our users don't want to migrate to the new Kiwi for this reason.

**Comment by westor7** [comment]
Yeah that one would be perfect if could be added, at least **60%+** loved the previously Tabs, users doesn't upgrade on the new kiwi because they mostly used tabs in the old version. let's hope for this addition soon 😄

**Comment by mzelenka** [comment]
Hey guys, is there any progression on tabs on top? Its harder to switch conversations on mobile devices and my users prefer older version.  Also chat room is alot easier to read because of text width.

Same as others in topic, my users dont want to migrate to newer because of that (i did ask alot of them).
Thanks.

**Comment by ivptr** [comment]
Any update on this? Our users still prefer the old client, because tabs are missing in the new one.
